### DISCUSSION TITLE
iPlayer All4 detect vpn

### DISCUSSION THREAD

**Discussion by sidelong** [Ysurac/openmptcprouter#2668] [General]
I have just succeeded in getting OMR working. It works well and the documentation is superb. First real issue is that All4 and iPlayer, and likely other such applications, wont work. They refuse to play, All4 throwing up an error message referring to the use of a VPN. I have searched the documentation and found references to bypassing OMR to enable Netflix use. Watching All4 and iPlayer in the UK is perfectly legitimate, so how should OMR be set up to facilitate this? Thanks in advance.

**Comment by Ysurac** [comment]
I don't know how All4 or iPlayer work, but you can first try to bypass domain in OMR-ByPass
